frontend/src/index.tsx

```diff
@@ -29,34 +26,15 @@ const cache = new InMemoryCache({
 const client = new ApolloClient({
   uri:
     process.env.REACT_APP_GRAPHQL_CLIENT_URI === undefined
```

Comment:
is there really env varaibles available in frontend?

Reply:
Yes, for local development purposes. When making changes to the frontend only, we can set this env var to the backend running on the dev server and avoid spinning up a local backend server

Reply:
Yes. TL;DR we have a workflow to make this available in the frontend.

My understanding is that when we build the Docker images for `smile-dashboard`, we first run a bash script that exports the variable `REACT_APP_GRAPHQL_CLIENT_URI` to the host machine's environment, then our `docker-compose` file specifies adding this variable to the Docker container's own environment. 

By [default](https://create-react-app.dev/docs/adding-custom-environment-variables/), the Create React App build system can embed environment variable names with the `REACT_APP_` prefix to the application.

In the future, we'll move away from running this script and just use a `.env` file for storing environment variables.

Reply:
thanks for this explaination